src/libsystemd-network/sd-dhcp-lease.c

```diff
@@ -735,6 +751,13 @@ int dhcp_lease_parse_options(uint8_t code, uint8_t len, const void *option, void
                         log_debug_errno(r, "Failed to parse captive portal, ignoring: %m");
                 break;
 
+        case SD_DHCP_OPTION_IPV6_ONLY_PREFERRED:
```

Comment:
The RFC said,
> If the client did not include the IPv6-Only Preferred option code in the Parameter Request List in the DHCPDISCOVER or DHCPREQUEST message, it MUST ignore the IPv6-Only Preferred option in any messages received from the server.

So, the option should be ignored if the IPv6 only mode is disabled.

Reply:
I see. OK.